src/js/stores/CosmosPackagesStore.js

```diff
@@ -263,6 +280,15 @@ class CosmosPackagesStore extends GetSetBaseStore {
     return null;
   }
 
+  getPackageVersions() {
+    const packageVersions = this.get("packageVersions");
+    if (packageVersions) {
+      return new UniversePackage(packageVersions);
```

Comment:
I personally don't think `UniversePackage` wrapper is needed here and we can just return the object directly.